plugins/services/src/js/service-configuration/PodStorageConfigSection.js

```diff
@@ -26,11 +25,11 @@ class PodStorageConfigSection extends React.Component {
         heading: 'Read Only',
         prop: 'readOnly',
         render(prop, row) {
-          return (
-            <ConfigurationMapBooleanValue
-              options={BOOLEAN_OPTIONS}
-              value={row[prop]} />
-          );
+          if (row[prop]) {
+            return BOOLEAN_OPTIONS.truthy;
```

Comment:
Yes, we want the strings here because we just want to show the words "TRUE" or "FALSE" in the UI.